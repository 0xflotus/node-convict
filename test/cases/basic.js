exports.conf = {
  env: {
    format: 'string ["production", "local"] = "local"',
    env: "NODE_ENV",
    doc: "The environment that we're running in."
  },
  URL: {
    format: 'string = "https://browserid.org"',
    env: "URL",
    doc: "The externally visible url of the server",
  },
  use_minified_resources: {
    format: "boolean = false;",
    doc: "All resources should be combined and minified",
    env: "MINIFIED"
  },
  var_path: {
    format: 'string = "/home/browserid/var"',
    doc: "The path the the 'var' directory, where logs and such will go"
  }
};
