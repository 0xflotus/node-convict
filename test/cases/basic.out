{
    "env":"local",
    "URL":"https://browserid.org",
    "var_path":"/home/browserid/var"
}
